src/main/java/me/earth/phobot/modules/combat/AutoTotem.java

```diff
@@ -5,19 +5,30 @@
 import me.earth.phobot.services.SurroundService;
 import me.earth.phobot.services.inventory.InventoryContext;
 import me.earth.phobot.util.entity.EntityUtil;
+import me.earth.pingbypass.api.input.Key;
+import me.earth.pingbypass.api.input.Keys;
 import me.earth.pingbypass.api.module.impl.Categories;
 import me.earth.pingbypass.api.setting.Setting;
 import me.earth.pingbypass.commons.event.SafeListener;
 import me.earth.pingbypass.commons.event.loop.GameloopEvent;
 import net.minecraft.client.multiplayer.ClientLevel;
 import net.minecraft.client.multiplayer.MultiPlayerGameMode;
 import net.minecraft.client.player.LocalPlayer;
+import net.minecraft.world.item.AxeItem;
+import net.minecraft.world.item.Item;
 import net.minecraft.world.item.Items;
+import net.minecraft.world.item.SwordItem;
 
 public class AutoTotem extends PhobotModule {
+    private final Setting<Offhand> mode = constant("Mode", Offhand.None, "What item you prefer in your offhand.");
```

Comment:
Hmmm, dont like that this is a setting. I would rather like a system with a bind to switch to a certain mode. I might make that myself at some point. Ok lets be real never, Cause AutoCrystal already switches to crystals for me.

Reply:
> Hmmm, dont like that this is a setting. I would rather like a system with a bind to switch to a certain mode. I might make that myself at some point. Ok lets be real never, Cause AutoCrystal already switches to crystals for me.

but what if you arent using phobot for ca, its just an optional setting, None -> It stays the same, as it is rn, but with sword gap and 2 others are just to make it always be totem or crystal